clients/client/src/main/java/org/projectnessie/client/http/impl/RequestContextImpl.java

```diff
@@ -99,4 +99,9 @@ public void addResponseCallback(BiConsumer<ResponseContext, Exception> responseC
   public List<BiConsumer<ResponseContext, Exception>> getResponseCallbacks() {
     return responseCallbacks;
   }
+
+  @Override
+  public String toString() {
+    return method.toString() + " to " + uri;
```

Comment:
nit: Do we really need `to`?.. `POST http://nessie/api/v1/...` would look nicer IMHO :)